### PR TITLE
CB-5051 handle internal calls with internal actor correctly

### PR DESCRIPTION
**Pull request by horadla23**

### CODE REVIEW

**Comment by horadla23** [comment]
- remove duplicated INTERNAL_ACTOR fields
- moved stastic internal actor field to ThreadBasedUserCrnProvider
- introduce doAsInternalActor calls
- fix internal calls for tag and telemtry endpoint, which failed previously

File: environment/src/main/java/com/sequenceiq/environment/tags/v1/controller/AccountTagController.java [hunk 1]
```diff
@@ -71,6 +71,12 @@ public AccountTagController(AccountTagService accountTagService,
     @CheckPermissionByAccount(action = AuthorizationResourceAction.ENVIRONMENT_READ)
     public AccountTagResponses list() {
         String accountId = ThreadBasedUserCrnProvider.getAccountId();
+        return listInAccount(accountId);
+    }
+
+    @Override
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.ENVIRONMENT_READ)
```
Comment (lnardai) [inline environment/src/main/java/com/sequenceiq/environment/tags/v1/controller/AccountTagController.java:78]: How is this limited to internalOnly?

**Comment by horadla23** [comment]
- introduced @InternalOnly annotation to mark api which should call with internal actor only

File: environment/src/main/java/com/sequenceiq/environment/tags/v1/controller/AccountTagController.java (hunk 1 above)
Comment (horadla23) [reply environment/src/main/java/com/sequenceiq/environment/tags/v1/controller/AccountTagController.java]: introduced an annotation for that, let's discuss if there is a simpler way to do this

**Pull request merged**